docs/examples/plugins/close-time.md

```diff
@@ -0,0 +1,61 @@
+### Request
+
+```javascript
+{
+  "idPath": "mappedObjects[].id",
+  "token": "YOUR_TOKEN_HERE",
+  "config": {
+  	  "idPath": "mappedObjects[].id",
```

Comment:
Die idPaths sind hier und in den anderen examples nicht immer gleich. Also an der Stelle und was die Einrückung angeht